run/development/README.md

```diff
@@ -0,0 +1,43 @@
+# Development stack
+
+> ⚠️ **Parabol does not provide any support on this stack**. Use it under your own resposibility.
+
+## General notes
+
+- **This stack is not meant for production use.** It is our development stack and can change at any moment, have errors and incorporate and remove components we are testing without any notice.
+- This stack is designed to be managed using `yarn db:start` and `yarn db:stop` to start the databases and `yarn dev` or `yarn start` to start and manage the application, depending on the use case.
```

Comment:
Changed. Now it says that the stack can be used either with `yarn dev` or building the application (there are more than one way to build it) and then running `yarn start`. Actually, it could also be used without any of those yarn options (with the scripts in the dist folder).